components/cam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1515,20 +1519,24 @@ subroutine aero_model_wetdep(dt, dlf, dlf2, cmfmc2, state,                    &!
     ncol  = state%ncol
 
     call physics_ptend_init(ptend, state%psetcols, 'aero_model_wetdep_ma', lq=wetdep_lq)
-    
+
     ! Do calculations of mode radius and water uptake if:
     ! 1) modal aerosols are affecting the climate, or
     ! 2) prognostic modal aerosols are enabled
-    
-    call t_startf('calcsize')
-    ! for prognostic modal aerosols the transfer of mass between aitken and accumulation
-    ! modes is done in conjunction with the dry radius calculation
-    call modal_aero_calcsize_sub(state, ptend, dt, pbuf)
-    call t_stopf('calcsize')
+    ! If not using prognostic aerosol call the diagnostic version
 
-    call t_startf('wateruptake')
-    call modal_aero_wateruptake_dr(state, pbuf)
-    call t_stopf('wateruptake')
+    ! Calculate aerosol size distribution parameters
```

Comment:
May be we can add the timers back.... (`call t_startf('calcsize')` and `t_startf('wateruptake')`)

Reply:
Sure, I figure they weren't that important, but I'll add them back.

Reply:
Thanks! I was not so concerned about the "calsize_diag" calls but it is good to have timers for calcsize_sub and wateruptake.